### PR TITLE
Export LoggerRequest, Fix local logger README

### PR DESCRIPTION
**Pull request by cliveseldon**
* Allow the logger structs to have exported fields so the code can be used as a library by other projects
 * Fix README for local logger demo

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/616)

### CODE REVIEW

File: pkg/logger/types.go [hunk 1]
```diff
@@ -12,11 +12,11 @@ const (
...
 type LogRequest struct {
-	url         *url.URL
-	b           *[]byte
-	contentType string
-	reqType     LogRequestType
-	id          string
-	sourceUri   *url.URL
-	modelId     string
+	Url         *url.URL
+	B           *[]byte
```
Comment (yuzisun) [inline pkg/logger/types.go]: Better to rename to `Bytes` here ?

Comment (cliveseldon) [reply pkg/logger/types.go]: Done

**Pull request merged**